# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1258463529 18000
#      Tue Nov 17 08:12:09 2009 -0500
# Node ID 64270d3ad46985ad67c613772c53d0736334f756
# Parent  84199c9fc69c3335dab7e6a5c4b26f738f78e484
mkf77def.in: only process *.f files

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,12 @@
+2009-11-17  John W. Eaton  <jwe@octave.org>
+
+	* mkf77def.in: Only process files with names that match *.f.
+
 2009-11-13  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am: Restore rules for building cruft.def.
 	(libcruft_la_LDFLAGS): Also append @XTRA_CRUFT_SH_LDFLAGS@.
 	(libcruft_la_DEPENDENCIES): New variable.
 	* mkf77def.in: Use @VAR@ instead of %VAR% substitutions.
 	Process	srcdir plus list of files instead of working as a filter.
 
diff --git a/libcruft/mkf77def.in b/libcruft/mkf77def.in
--- a/libcruft/mkf77def.in
+++ b/libcruft/mkf77def.in
@@ -46,10 +46,14 @@ fi
 if [ $# -gt 1 ]; then
   srcdir="$1"
   shift
 fi
 
 echo EXPORTS
 for arg
 do
-  $SED -n -e 'y/ABCDEFGHIJLKMNOPQRSTUVWXYZ/abcdefghijlkmnopqrstuvwxyz/; s/^\(      \|	\)[ 	]*\(.*function\|subroutine\|entry\)[ 	]*\([^ 	(]*\).*$/\3/p' "$srcdir/$arg" | eval $awkcmd
+  case "$arg" in
+    *.f)
+      $SED -n -e 'y/ABCDEFGHIJLKMNOPQRSTUVWXYZ/abcdefghijlkmnopqrstuvwxyz/; s/^\(      \|	\)[ 	]*\(.*function\|subroutine\|entry\)[ 	]*\([^ 	(]*\).*$/\3/p' "$srcdir/$arg" | eval $awkcmd
+    ;;
+  esac
 done
